packages/react-charting/src/components/CommonComponents/CartesianChart.base.tsx

```diff
@@ -463,7 +465,14 @@ export class CartesianChartBase extends React.Component<IModifiedCartesianChartP
         ref={(rootElem: HTMLDivElement) => (this.chartContainer = rootElem)}
         onMouseLeave={this._onChartLeave}
       >
-        <FocusZone direction={focusDirection} className={this._classNames.chartWrapper} {...svgFocusZoneProps}>
+        {!this._isFirstRender && <div id={this.idForDefaultTabbableElement} />}
+        <FocusZone
+          direction={focusDirection}
+          className={this._classNames.chartWrapper}
+          defaultTabbableElement={`#${this.idForDefaultTabbableElement}`}
```

Comment:
Can we set defaultTabbableElement as null in second render cycle?

Reply:
I tried this , but this does not for a few charts , also I saw Focus Zone code the logic to updateTabIndexes gets triggered only when there is a defaultTabbableElement and then it is no longer inside FocusZone element([code](https://github.com/ankityadav4/fluentui/blob/4b802ba3745dde01c68224329c1414f077faa1db/packages/react-focus/src/components/FocusZone/FocusZone.tsx#L244))